app/scenes/Settings/ImportExport.js

```diff
@@ -199,6 +211,46 @@ function ImportExport() {
           ? `${t("Requesting Export")}…`
           : t("Export Data")}
       </Button>
+      <Subheading>
+        <Trans>Recent exports</Trans>
+      </Subheading>
+      <PaginatedList
+        items={fileOperations.orderedDataExports}
+        fetch={fileOperations.fetchPage}
+        options={{ type: "export" }}
+        renderItem={(item) => (
+          <ListItem
+            key={item.id}
+            title={
+              item.collection ? item.collection.name : t("All collections")
+            }
+            subtitle={
+              <>
+                <Flex>
+                  {stateMapping[item.state]}&nbsp;•&nbsp;
+                  {user.id === item.user.id ? t("You") : item.user.name}{" "}
+                  created&nbsp;
+                  <Time dateTime={item.createdAt} addSuffix shorten />
+                  &nbsp;•&nbsp;{item.sizeInMB}
+                </Flex>
+              </>
+            }
+            actions={
+              <Button
+                onClick={() =>
+                  window.open(
+                    `/api/fileOperations.redirect?id=${item.id}`,
+                    "_blank"
+                  )
+                }
+                neutral
+              >
+                {t("Download")}
+              </Button>
```

Comment:
I _believe_ this will work

```suggestion
              <Button
                as="a"
                href={`/api/fileOperations.redirect?id=${item.id}`}
                neutral
              >
                {t("Download")}
              </Button>
```